crates/dbsp/src/circuit/runtime.rs

```diff
@@ -774,7 +774,7 @@ impl Runtime {
                     .unwrap_or({
                         // This reduces the files stored on disk to a reasonable number.
 
-                        1024 * 1024
+                        10 * 1024 * 1024
```

Comment:
Please also update crates/feldera-types/src/config.rs to say that 10 MiB is the default. Currently it says 1 MiB:

```
    /// A value of 0 will write even empty batches to storage, and nonzero
    /// values provide a threshold.  `usize::MAX` would effectively disable
    /// storage for such batches.  The default is 1,048,576 (1 MiB).
```